source/html/providers.html

```diff
@@ -27,16 +27,18 @@
               <h1 class="pf-c-title pf-m-3xl">Log in with&hellip;</h1>
               <ul>
                 {{ range $provider := .Providers }}
-                  <li class="idp">
-                    <a href="{{$provider.URL}}" class="pf-c-button pf-m-secondary pf-m-block" title="Log in with {{$provider.Name}}">{{$provider.Name}}</a>
-                  </li>
+                  {{ if ne $provider.Name "kube:admin" }}
+                    <li class="idp">
+                      <a href="{{$provider.URL}}" class="pf-c-button pf-m-secondary pf-m-block" title="Log in with {{$provider.Name}}">{{$provider.Name}}</a>
+                    </li>
+                  {{ end }}
```

Comment:
Source for this in the [README](https://github.com/openshift/managed-cluster-config/tree/master/source/html) is https://openshift.github.io/oauth-templates/od/providers.html which doesn't include this.  Recommend we get that updated upstream.

Reply:
Added follow up item to upstream this fix